recipes/hapnest/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: hapnest
+  version: "1"
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - julia >=1.6.4
+
+source:
+  url: https://github.com/intervene-EU-H2020/synthetic_data/archive/refs/heads/main.zip
```

Comment:
can you maybe tag a release and use that instead of `main`

Reply:
There are no releases available